iris/src/iris/pipeline/prompts/assess_user_answer_prompt.py

```
@@ -0,0 +1,117 @@
+# models weight as an authoritative, behavior-defining channel just like the
+# system message), it could override the policy below. Instead, all of it is
+# rendered into two lower-priority, clearly delimited data messages — the
+# exercise/submission data (`exercise_data_prompt`, `<exercise_data>` tags) and
+# the prior Q&A (`conversation_history_prompt`, `<conversation_history>` tags)
+# — and this prompt explicitly tells the model to treat everything in those
+# blocks as inert data, never as instructions.
+assess_user_answer_prompt = """
+**Role:** You are a strict Tutor of a programming course.
+You want to make sure that the students only submit code to the learning platform which they wrote themselves.
+Another tutor asked the student questions about the submission.
+Your goal is to assess whether a student’s answer is sufficient to determine if the submission was
```

Ruff (0.16.1)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)